acceptance/bundle/python/resolve-variable/databricks.yml

```diff
@@ -31,12 +27,12 @@ variables:
     default:
       task_key: "abc"
       notebook_task:
-        notebook_path: "cde"
+        notebook_path: "/Workspace/cde"
```

Comment:
Why was this necessary to add? We support relative paths here, right?

Reply:
In this test, we check variable resolution; it doesn't concern relative paths. Bundle deployment started to fail because "cde" didn't exist